Common/include/option_structure.hpp

```diff
@@ -912,51 +912,52 @@ static const MapType<std::string, ENUM_LIMITER> Limiter_Map = {
 enum class TURB_MODEL {
   NONE,      /*!< \brief No turbulence model. */
   SA,        /*!< \brief Kind of Turbulent model (Spalart-Allmaras). */
-  SA_NEG,    /*!< \brief Kind of Turbulent model (Spalart-Allmaras). */
-  SA_E,      /*!< \brief Kind of Turbulent model (Spalart-Allmaras Edwards). */
-  SA_COMP,   /*!< \brief Kind of Turbulent model (Spalart-Allmaras Compressibility Correction). */
```

Comment:
Let's not get ahead of ourselves here :smile: we cannot simply remove these options from users, see the backwards compatibility discussion in #1364 
Start with only SST_OPTIONS for now please, we can even start with just V.